### DISCUSSION TITLE
Change word diff colour

### DISCUSSION THREAD

**Discussion by srasku** [dandavison/delta#1067] [Q&A] [answered]
Is it possible to change the internal word diff colour?  In this example, the line diff colour background is blue and the word diff is green.  I actually wanted the reverse.  Is it possible?

<img width="910" alt="green each" src="https://user-images.githubusercontent.com/2049005/166481803-30417bcb-c12b-4dde-9555-8ca4a04e9a12.png">

For reference, this is my `delta` configuration:

```
[interactive]
	diffFilter = delta --color-only
[delta]
	navigate = true
	line-numbers = true
	file-style = "#ffff00" bold
	plus-style = syntax blue
[merge]
	conflictstyle = diff3
```

**Comment by dandavison** [comment, marked as answer]
Hi @srasku, yes, what you're calling the "word diff style" is `plus-emph-style`. There's also `minus-style` (dark red in your screenshot) and `minus-emph-style` (bright red in your screenshot). The full documentation of all styles is in `delta --help`.
